lua/pl/path.lua

```diff
@@ -80,10 +80,10 @@ end
 --- Get the working directory.
 -- Implicit link to [`luafilesystem.currentdir`](https://keplerproject.github.io/luafilesystem/manual.html#reference)
 -- @function currentdir
-path.currentdir = function(d)
-  local ok, err, code = currentdir(d)
+path.currentdir = function()
+  local ok, err, code = currentdir()
   if not ok then
-    return ok, err_func("currentdir", d, err, code), code
+    return ok, err_func("currentdir", "", err, code), code
```

Comment:
```suggestion
    return ok, err_func("currentdir", nil, err, code), code
```
Either this should be `nil` I think, or we should adjust the `err_func` implementation to deal with a variable number of arguments.